infra-as-code/bicep/orchestration/mgDiagSettingsAll/README.md

```diff
@@ -0,0 +1,147 @@
+  - **Required**
+  - Deploys following child Landing Zone Management groups if set to `true`:
+    - `Confidential Corp`
+    - `Confidential Online`
+- `parLandingZoneMgChildren`
+  - Object - default is an empty array `[]`
+  - **Optional**
+  - Deploys whatever you specify in the object as child Landing Zone Management groups.
+
+#### `parLandingZoneMgChildren` Input Examples
+
+Below are some examples of how to use this input parameter in both Bicep & JSON formats.
+
+##### Bicep Example
+
+```bicep
+parLandingZoneMgChildren: {
+  pci: {
+    displayName: 'PCI'
+  }
+  'another-example': {
+    displayName: 'Another Example'
+  }
+}
+```
+
+##### JSON Parameter File Input Example
+
+```json
+    "parLandingZoneMgChildren": {
+      "value": [
+        "pci",
+        "avs"
+    ]}
+```
+
+## Outputs
+
+The module will not generate any outputs.
+
+## Deployment
+
+In this example, the Diagnostic Settings are enabled on the management groups through a tenant-scoped deployment.
+
+> For the examples below we assume you have downloaded or cloned the Git repo as-is and are in the root of the repository as your selected directory in your terminal of choice.
+
+### Azure CLI
+
+```bash
+# For Azure global regions
+az deployment tenant create \
+  --template-file infra-as-code/bicep/orchestration/mgDiagSettingsAll/mgDiagSettingsAll.bicep \
+  --parameters @infra-as-code/bicep/orchestration/mgDiagSettingsAll/parameters/mgDiagSettingsAll.parameters.all.json \
+  --location eastus
+```
+
+OR
+
+```bash
+# For Azure China regions
+az deployment tenant create \
+  --template-file infra-as-code/bicep/orchestration/mgDiagSettingsAll/mgDiagSettingsAll.bicep \
+  --parameters @infra-as-code/bicep/orchestration/mgDiagSettingsAll/parameters/mgDiagSettingsAll.parameters.all.json \
+  --location chinaeast2
+```
+
+### PowerShell
+
+```powershell
+# For Azure global regions
+New-AzTenantDeployment `
+  -TemplateFile infra-as-code/bicep/orchestration/mgDiagSettingsAll/mgDiagSettingsAll.bicep `
+  -TemplateParameterFile infra-as-code/bicep/orchestration/mgDiagSettingsAll/parameters/mgDiagSettingsAll.parameters.all.json `
+  -Location eastus
+```
+
+OR
+
+```powershell
+# For Azure China regions
+New-AzTenantDeployment `
+  -TemplateFile infra-as-code/bicep/orchestration/mgDiagSettingsAll/mgDiagSettingsAll.bicep `
+  -TemplateParameterFile infra-as-code/bicep/orchestration/mgDiagSettingsAll/parameters/mgDiagSettingsAll.parameters.all.json `
+  -Location chinaeast2
+
+```
+
+## Validation
+
+To validate if Diagnostic Settings was correctly enabled for any specific management group, a REST API GET call can be used. Documentation and easy way to try this can be found in this link [(Management Group Diagnostic Settings - Get)](https://learn.microsoft.com/rest/api/monitor/management-group-diagnostic-settings/get?tabs=HTTP&tryIt=true&source=docs#code-try-0). There is currently not a direct way to validate this in the Azure Portal, Azure CLI or PowerShell.
```

Comment:
Add a note here as to why this is required. Because the portal experience does not show these 👍